OpenRA.Mods.Common/Traits/Carryall.cs

```diff
@@ -192,6 +192,12 @@ IEnumerable<IRenderable> IRender.Render(Actor self, WorldRenderer wr)
 			}
 		}
 
+		Rectangle IRender.AutoRenderBounds(Actor self)
+		{
+			// Actor.RenderBounds unions all non-empty sprite bounds, so this wouldn't have an effect on the final bounds anyway as RenderSprites/RenderBounds take care of that
```

Comment:
Correct me if i'm wrong, but `RenderSprites` doesn't know anything about the carried actor's renderables, so it won't account for those when returning the carryall's own render bounds.  This will need to aggregate over the previewRenderable's bounds in the same way that `RenderSprites` will.